src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    public String color;
+    public int number;
```

Comment:
Fields should be private. Set them up using the constructor with arguments OR setters